### PR TITLE
DOCS-3659 Browser Monitoring Versioning Edits

### PR DESCRIPTION
**Pull request by alai97**
## Motivation

DOCS-3659

## Changes

Adds version tabs for latest + before 4.10.12 for browser sessions.

## Preview

https://docs-staging.datadoghq.com/alai97/browser-monitoring-page-versions/real_user_monitoring/browser/

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

**Comment by MirandaKapin** [comment]
@BenoitZugmeyer The challenge is that customers are still confused about which version of the SDK supports which configuration type. Meaning that it's still being overlooked by customers - The idea with adding the tabs is that people will need to think about which version of the SDK they are using

**Comment by MirandaKapin** [comment]
@BenoitZugmeyer remind me - which version of the SDK did we introduce the replaySampleRate?

**Review by buraizu — APPROVED** [review]

**Comment by BenoitZugmeyer** [comment]
@MirandaKapin 
> @BenoitZugmeyer remind me - which version of the SDK did we introduce the replaySampleRate?
v3.0.0

File: packages/rum/README.md [hunk 1]
```diff
@@ -48,17 +71,48 @@ datadogRum.init({
...
 ```
+</details>
 
 The `trackInteractions` and `trackFrustrations` parameters enable the automatic collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
 
 ### CDN async
 
 Add the generated code snippet to the head tag of every HTML page you want to monitor in your application.
 
+<details open="false">
```
Comment (alai97) [inline packages/rum/README.md:86]:
```suggestion
<details open>
```

File: packages/rum/README.md [hunk 2]
```diff
@@ -91,6 +146,31 @@ Early RUM API calls must be wrapped in the `DD_RUM.onReady()` callback. This ens
 
 Add the generated code snippet to the head tag (in front of any other script tags) of every HTML page you want to monitor in your application. Including the script tag higher and synchronized ensures Datadog RUM can collect all performance data and errors.
 
+<details open="false">
```
Comment (alai97) [inline packages/rum/README.md:149]:
```suggestion
<details open>
```

File: packages/rum/README.md [hunk 3]
```diff
@@ -91,6 +146,31 @@ Early RUM API calls must be wrapped in the `DD_RUM.onReady()` callback. This ens
...
+      sampleRate: 100,
+      premiumSampleRate: 100, // if not included, the default is 100
+      trackInteractions: true,
+    })
+</script>
+```
+</details>
+
+<details>
+  <summary><code>v4.10.2</code> and before</summary>
```
Comment (alai97) [inline packages/rum/README.md:172]:
```suggestion
  <summary>before<code>v4.10.2</code></summary>
```

File: packages/rum/README.md [hunk 4]
```diff
@@ -307,33 +387,81 @@ Two initialization parameters are available to control how the session is tracke
 
 To track 100% of your sessions as Browser:
 
+<details open="false">
```
Comment (alai97) [inline packages/rum/README.md:390]:
```suggestion
<details open>
```

File: packages/rum/README.md [hunk 5]
```diff
@@ -307,33 +387,81 @@ Two initialization parameters are available to control how the session is tracke
...
 datadogRum.init({
     ....
     sampleRate: 100,
     premiumSampleRate: 0
 });
 ```
+</details>
+
+<details>
+  <summary><code>v4.10.2</code> and before</summary>
```
Comment (alai97) [inline packages/rum/README.md:403]:
```suggestion
  <summary>before<code>v4.10.2</code></summary>
```

File: packages/rum/README.md [hunk 6]
```diff
@@ -307,33 +387,81 @@ Two initialization parameters are available to control how the session is tracke
...
 datadogRum.init({
     ....
     sampleRate: 100,
     premiumSampleRate: 100
 });
 ```
+</details>
+
+<details>
+  <summary><code>v4.10.2</code> and before</summary>
```
Comment (alai97) [inline packages/rum/README.md:429]:
```suggestion
  <summary>before<code>v4.10.2</code></summary>
```

File: packages/rum/README.md [hunk 7]
```diff
@@ -307,33 +387,81 @@ Two initialization parameters are available to control how the session is tracke
...
 datadogRum.init({
     ....
     sampleRate: 60,
     premiumSampleRate: 50
 });
 ```
+</details>
+
+<details>
+  <summary><code>v4.10.2</code> and before</summary>
```
Comment (alai97) [inline packages/rum/README.md:455]:
```suggestion
  <summary>before<code>v4.10.2</code></summary>
```

File: packages/rum/README.md [hunk 8]
```diff
@@ -307,33 +387,81 @@ Two initialization parameters are available to control how the session is tracke
...
+    ....
+    sampleRate: 100,
+    replaySampleRate: 100
+});
+```
+</details>
 
 The `premiumSampleRate` is a percentage of `sampleRate`. If you set `sampleRate` to 60 and `premiumSampleRate` to 50, 40% of sessions are dropped, 30% of sessions are collected as Browser and 30% of sessions are collected as Browser Premium.
 
+<details open="false">
```
Comment (alai97) [inline packages/rum/README.md:442]:
```suggestion
<details open>
```

File: packages/rum/README.md [hunk 9]
```diff
@@ -48,17 +71,48 @@ datadogRum.init({
...
+      premiumSampleRate: 100, // if not included, the default is 100
+      trackInteractions: true,
+    })
+  })
+</script>
+```
+</details>
+
+<details>
+  <summary><code>v4.10.2</code> and before</summary>
```
Comment (alai97) [inline packages/rum/README.md:114]:
```suggestion
  <summary>before<code>v4.10.2</code></summary>
```

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**